### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by kmcgill88** [kmcgill88/admob_flutter#76]
To reduce bugs we desperately need to add CI. Even a basic project build would be helpful.

I propose we add Travis CI or similar to improve project quality!

Suggested Build steps.
1. Dart Code needs to be unit tested
1. Native code needs unit tests
1. iOS and Android projects need to compile
1. (stretch) Compile projects on multiple versions. ie. iOS 11, 12, 13 and Android 8,9,10

**Comment by kmcgill88** [comment]
@YoussefKababe @EliasDeuss do either of you have experience doing this? I've only used Travis CI with Github. I'm happy to set it up.

@YoussefKababe I would need repo permission to integrate with Travis CI. This is my setup on a different project of mine, [McPicker](https://github.com/kmcgill88/McPicker-iOS/blob/master/.travis.yml)

**Comment by kmcgill88** [comment]
@YoussefKababe would you be willing to give me repo permission to set this up?

**Comment by EliasDeuss** [comment]
> @YoussefKababe @EliasDeuss do either of you have experience doing this? I've only used Travis CI with Github. I'm happy to set it up.

No I have only used Travis for a flutter project, not for a plugin.

**Comment by kmcgill88** [comment]
@YoussefKababe @EliasDeuss check out the build I started on [bitrise](https://app.bitrise.io/app/aab4844dfe687df4#/builds). I also added a build badge to the read me.

Current build steps:
1. `flutter packages get`
1. `flutter analyze`
1. `flutter build ios --debug --no-codesign`

I'd like to see any changes unit tested and we add `flutter test` as well as make sure things compile for Android with `flutter build apk --debug --no-codesign`

More to come!
